### DISCUSSION TITLE
Is there a possibility to get the game display into a numpy array ?

### DISCUSSION THREAD

**Discussion by Fredovsky** [kitao/pyxel#496] [Q&A]
I am developing games for my custom black&white screen.

I need to send image data in the form of a numpy array. Is there a possibility to get the image screen data into a numpy array using Pyxel ?

Thanks

**Comment by uu-nn** [comment]
```
import ctypes
import numpy

my_img_pointer = img.data_ptr() # <- your image
my_img_numpy = np.ctypeslib.as_array(my_img_pointer) # <- numpy array
```
